### DISCUSSION TITLE
help with transition from pip to pipx because of exernally-managed-environment error

### DISCUSSION THREAD

**Discussion by ghost** [pypa/pipx#1035] [Q&A] [answered]
I had a program installed with pip before they broke it with that exernally-managed-environment error. So my only choices are that break-system option which sounds scary and install the program again with pipx. 

But when I installed it again with pipx it's using the program from pip instead, so it's an old version I can't update any longer. 

Because of the externally-managed-environment error I can't uninstall the program with pip.

What's the best way in your opinion to fix this?

**Comment by uranusjr** [comment, marked as answer]
You can delete the executable manually with `rm` before installing it with pipx. With pip >=23.2 you can also use `--break-system-packages` to forcefully uninstall the package.
